crates/cli/src/main.rs

```diff
@@ -224,6 +228,14 @@ fn main() -> Result<(), anyhow::Error> {
                     sub_matches.get_one::<usize>("max-parallelism").cloned();
                 let maybe_request_wait_timeout =
                     sub_matches.get_one::<u64>("request-wait-timeout").cloned();
+                let static_patterns = sub_matches
+                    .get_many::<String>("static")
+                    .unwrap()
+                    .map(|s| s.as_str())
+                    .collect::<Vec<&str>>();
```

Comment:
When the `--static` option is not given by the user, the process dies by panic. Is this intended?